src/js/components/stock-movement-wizard/outbound/AddItemsPage.jsx

```diff
@@ -379,6 +392,20 @@ class AddItemsPage extends Component {
         key => key !== 'product',
       );
 
+      if (newQty === oldQty && oldItem.rowSaveStatus === RowSaveStatus.ERROR) {
```

Comment:
I don't understand this part. Does it mean, that if the newly entered quantity equals the previous state quantity and if it was errored in the previous state, then now we want to mark it as saved?

Reply:
It means that, if we set wrong quantity (less than 0) our row has `ERROR` status, when we set the same quantity as we had before the `ERROR` status, the request isn't send, so I am checking here if we have the same quantity as before (because this value is currently saved - that's the reason why I set `SAVED` status). When we changed the value to other than the previous one the request will be send, and we will have `SAVED` status after getting response.